webpack.dev.babel.js

```diff
@@ -9,7 +9,8 @@ import webpack from 'webpack';
 export default {
   entry: {
     app: [
-      'webpack/hot/only-dev-server',
+      'webpack-dev-server/client?http://localhost:8080',
```

Comment:
Mind changing the loopback to `0.0.0.0` instead of localhost? Using `localhost` here will cause the dev server to bomb when run inside a docker container.